zipline/data/us_equity_pricing.py

```diff
@@ -474,7 +497,20 @@ def _table(self):
 
     @lazyval
     def _calendar(self):
```

Comment:
Should we rename this to `sessions` or something like it?  It's confusing to have both `_calendar` and `trading_calendar`.